SparkPost/2web2ui
# FE-1143 - [Dashboard UX Redesign] Incorporate Hibana-only footer in the app

[FE-1143](https://jira.int.messagesystems.com/browse/FE-1143)

### What Changed
- Adds new Hibana-only `<Footer/>` component
- Adds supporting Cypress tests
- Used `styled-components` for the first time! We need to chat about this this afternoon.
- Updates design tokens package for Hibana only - while working on this feature, I noticed we were on an outdated version
- **Note:** Test failure due to flakiness, addressed in #1723 

### How To Test
- Log in to the app, turn on Hibana, and verify the links render as expected and open in new tabs.
- Resize the browser slowly and verify it remains in alignment with content above it horizontally
- Verify that the design roughly matches the mocks (**Note:** I did have to make the text color darker to meet a11y standards and use our `<ExternalLink/>` component here for consistent behavior across the app)

### To Do
- [x] Wait on meeting on 7/28/20 before review/merging
